File: scratch/list_sections.js
const fs = require('fs');
const path = require('path');

const filePath = 'c:\\yora\\yora\\src\\app\\home\\page.tsx';
const content = fs.readFileSync(filePath, 'utf8');
const lines = content.split('\n');

lines.forEach((line, index) => {
  const match = line.match(/(?:\{\/\*|\/\/)\s*(\d+\..*?)(?:\*\/\}|$)/);
  if (match) {
    console.log(`Line ${index + 1}: ${match[1].trim()}`);
  }
});

File: scratch/list_sections.py
import re

file_path = r"c:\yora\yora\src\app\home\page.tsx"
with open(file_path, "r", encoding="utf-8") as f:
    lines = f.readlines()

for i, line in enumerate(lines, 1):
    # Match comments like {/* 1. Announcement Bar */} or // 1. ...
    match = re.search(r'(?:\{/\*|//)\s*(\d+\..*?)(?:\*/\}|$)', line)
    if match:
        print(f"Line {i}: {match.group(1).strip()}")
